tensorflow/compiler/mlir/tosa/transforms/tfl_legalize_patterns.td

```diff
@@ -21,15 +21,9 @@ include "tensorflow/compiler/mlir/lite/ir/tfl_ops.td"
 include "mlir/Dialect/Quant/QuantOps.td"
 include "mlir/Dialect/Tosa/IR/TosaOps.td"
 
+//===----------------------------------------------------------------------===//
 // Unary ops patterns.
-def : Pat<(TFL_AbsOp $arg), (Tosa_AbsOp $arg)>;
```

Comment:
Don't delete these, you can specify names directly in tablegen, e.g.

def ConvertTFLAbsOp : Pat<(TFL_AbsOp $arg), (Tosa_AbsOp $arg)>;

Tablegen substantially decreases the amount of C++ required for simple cases.

Reply:
We are using an OpConversion pattern with an Adaptor and templates. The amount of code should be similar.

Reply:
Done